### ISSUE TITLE
Outdoorsman 

### ISSUE THREAD

**Issue by Slagar80** [rotators/Fo1in2#246]
I know this really isn’t an issue but I don’t wanna bother lex alone with this and maybe some of you guys know a bit more about this.

What is the effect of the outdoorsman skill in Fallout et tu?

I know that Lex added an increase in traveling speed which indirectly decreases encounter change and if I’m not mistaken I saw in the encounter scripts of the fo1 base mod that it effects the probability of getting a cave map.

But according to soma wikis in Fallout it decreases the negative consequences of dehydration and rockfall encounters and in Fallout 2 increases the probability of getting an avoid encounter prompt (wich isn’t a thing in et tu I know)

https://fallout.fandom.com/wiki/Outdoorsman

According to another wiki it decreases the likelihood of bad encounters and increases the likelihood of special encounters in both Fallout and Fallout 2 and increases the traveling speed in Fallout (which like I said lex recently added)

https://fallout-archive.fandom.com/wiki/Outdoorsman

Since et tu uses the Fallout 2 engine I suppose it uses the Fallout 2 skill effects unless you guys change that in the fo1 base mod.

But aside from the prompt effect which is, like I said, not a thing in et tu, I’m under the impression that the likelihood changes for bad and special encounters also doesn’t work in et tu since et tu uses its own encounter scripts in the fo1 base mod and as far as I’m aware Outdoorsman isn’t a factor in the encounter rolls.

So long story short:
What does outdoorsman do in et tu aside from the recently implemented travel speed changes and are my above assumptions correct?

**Comment by Slagar80** [comment]
Sry for bothering you

I just found the entries in the mapenc.h that make outdoorsman effect the dehydration and rockfall encounters like in Fallout.

Nice job converting the skills functionality into the Fallout 2 engine.
